test/source/tests/compose.ts

```diff
@@ -873,7 +873,7 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
     }));
 
     ava.default('compose - enforce message signing when encrypting', testWithBrowser('compatibility', async (t, browser) => {
-      const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compose');
+      const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility');
```

Comment:
Unsure why this change

Reply:
well, if we pass `compose` to `ComposePageRecipe.openStandalone` method, compose.htm `account_email` becomes `ci.tests.gmail@flowcrypt.test` which is not correct (logged in account email is `flowcrypt.compatibility@gmail.com`)

https://github.com/FlowCrypt/flowcrypt-browser/blob/4736d924ef7d7454401f38037f6ea00eb0c39173/test/source/tests/page-recipe/compose-page-recipe.ts#L27

Reply:
Generally we use `compose` account for tests that compose emails, and `compatibility` email for tests that read / check emails. I guess that doesn't really matter in particular.

But I'm wondering why now? If the test was broken, it would have to be already failing before?

Reply:
It matters with new change and I can see that previous logic was incorrect.
Current logged in user email is `flowcrypt.compatibility@gmail.com` and test is trying to open compose.htm page with `account_email` as `ci.tests.gmail@flowcrypt.test`.
As a result, storage.rules becomes null because provided `account_email` is incorrect

![image](https://user-images.githubusercontent.com/77344191/192783537-623d3720-c27f-463c-bce9-1ebf5f498743.png)
![image](https://user-images.githubusercontent.com/77344191/192783750-bf17f41e-5286-4d76-aaa1-7177a6a92849.png)

Reply:
Previously tests didn't fail because we didn't throw error for invalid/empty rules from AcctStore.

Reply:
Any reason why we need to use `ci.tests.gmail@flowcrypt.test` for `compose` group?

![image](https://user-images.githubusercontent.com/77344191/192818467-15e5a0f5-c061-4e32-bba6-19c2399f39fa.png)

Reply:
No strong reason. It's generally preferred to use our own test email on a `.test` domain to make things clear. So we generally always try to use a `.test` domain, except when we cannot.

Many tests use exported data specifically from `flowcrypt.compatibility` account, and for those, we use the same account for simplicity. For tests that don't need those exported messages, like compose tests, we use the `.test` account.

It's not a big deal on its own. I'm just trying to get to the bottom of what happened above. I get it conceptually, I think, but something still smells :-) That's why I'd like to see what will it look like when using the original `compose` test email, and fixing whatever needed fixing. That will surface the needed change more clearly.

Reply:
Hmm... Not sure why you want that change..
Current app logic is to receive `acct_email` from url parameters.
`openStandalone` test method opens `compose.htm` page with incorrect `account_email` parameter.

You mean you want to change current app logic not to receive `acct_email` from url parameters? And receive `acct_email` from somewhere else

Reply:
What I meant is this:
 - openStandalone is passed with `compose` as before
 - that uses `ci.tests.gmail@flowcrypt.test` for this test as before

If that breaks the test, fix the breakage by fixing that test, without changing on which account it runs. It should continue to run on `ci.tests.gmail@flowcrypt.test`. If it doesn't now, fix whatever makes it not be able to run as before. Does that make sense?

Reply:
> It should continue to run on `ci.tests.gmail@flowcrypt.test`.

It used to run on `compatibility` account, that's the point. So we need to put `compose` (if it is accepted there) or `ci.tests.gmail` to line 875, I guess

Reply:
Ah, now I see the line 875 above, as you mentioned:

```
    ava.default('compose - enforce message signing when encrypting', testWithBrowser('compatibility', async (t, browser) => {
      const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility');
```

Yes, setting 875 to `ci.tests.gmail` and 876 to `compose` will work.

Sorry for the confusion, I didn't notice the mismatch before.

Reply:
Actually, running on compatibility is fine to. So I'm merging this. Thank you for explaining this to me.